chart/k8gb/values.yaml

```diff
@@ -10,6 +10,8 @@ k8gb:
   # image tag defaults to Chart.AppVersion, see Chart.yaml
   # but can be overrided with imageTag key
   # imageTag:
+  hookImageRepo: bitnami/kubectl
+  hookImageTag: 1.21.1
```

Comment:
Can we combine it in a single hookImage param? like `hookImage: bitnami/kubectl:1.21.1`